__tests__/app.test.js

```diff
@@ -79,43 +79,101 @@ describe('/api/articles', () => {
-                expect(msg).toBe('Not found')
+        test('GET 404 /api/articles/:article_id', () => {
+            return request(app)
+                .get('/api/articles/69')
+                .expect(404)
+                .then(({ body: { msg } }) => {
+                    expect(msg).toBe('Not found')
 
-            })
-    });
-});
+                })
+        });
+    })
+
+    describe('PATCH /api/articles/:article_id', () => {
+        test('PATCH 200 /api/articles/1', () => {
+            return request(app)
+                .patch('/api/articles/1')
+                .send({ inc_votes: 1 })
+                .expect(200)
+                .then(({ body: { article } }) => {
+                    expect(article.article_id).toBe(1)
+                    expect(article.title).toBe("Living in the shadow of a great man")
+                    expect(article.topic).toBe("mitch")
+                    expect(article.author).toBe("butter_bridge")
+                    expect(article.body).toBe("I find this existence challenging")
+                    expect(article.created_at).toBe("2020-07-09T20:11:00.000Z")
+                    expect(article.votes).toBe(101)
+                    expect(article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
+
+                })
+        })
+        test('PATCH 200 /api/articles/1', () => {
+            return request(app)
+                .patch('/api/articles/1')
+                .send({ inc_votes: -100 })
+                .expect(200)
+                .then(({ body: { article } }) => {
+                    expect(article.article_id).toBe(1)
+                    expect(article.title).toBe("Living in the shadow of a great man")
+                    expect(article.topic).toBe("mitch")
+                    expect(article.author).toBe("butter_bridge")
+                    expect(article.body).toBe("I find this existence challenging")
+                    expect(article.created_at).toBe("2020-07-09T20:11:00.000Z")
+                    expect(article.votes).toBe(0)
+                    expect(article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
+
+                })
+        })
+        test('PATCH 400 /api/articles/cat', () => {
+            return request(app)
+                .patch('/api/articles/cat')
+                .send({ inc_votes: 1 })
+                .expect(400)
+                .then(({ body: { msg } }) => {
+                    expect(msg).toBe('Invalid input')
+                })
+        });
+
+        test('PATCH 404 /api/articles/69', () => {
+            return request(app)
+                .patch('/api/articles/69')
+                .send({ inc_votes: 1 })
+                .expect(404)
+                .then(({ body: { msg } }) => {
+                    expect(msg).toBe('Not found')
+                })
+        });
+    })
+})
```

Comment:
Some tests to consider:

400 - inc_votes is missing
400 - inc_votes is not a number 

200 - ignores additional (potentially malicious) keys